pyvisa_py/serial.py

```diff
@@ -34,22 +34,11 @@
 IS_WIN = sys.platform == "win32"
 
 
-def iter_bytes(data: bytes, mask: Optional[int] = None, send_end: bool = False):
-    if send_end and mask is None:
-        raise ValueError("send_end requires a valid mask.")
-
-    if mask is None:
-        for d in data:
-            yield bytes([d])
-
-    else:
-        for d in data[:-1]:
-            yield bytes([d & ~mask])
-
-        if send_end:
-            yield bytes([data[-1] | ~mask])
-        else:
-            yield bytes([data[-1] & ~mask])
+def iter_bytes(
+    data: bytes, mask: Optional[int] = None, send_end: Optional[bool] = False
+):
```

Comment:
I wasn't sure if `serial.iter_bytes` was used by any external projects, so I opted to keep the API around. LMK if you'd rather I just remove it.